docs/move/usage.rst

```diff
@@ -0,0 +1,265 @@
+MoveProver Setup and Specification Guide
+========================================
+
+This guide explains how to set up Move specifications ("specs"), write rules,
+summaries, and advanced constructs for verifying Sui Move contracts with the
+Certora MoveProver.
+
+Setup
+-----
+
+Move “specs” (rules and summaries) are written in Move. Typically these live in
+their own Move package, often named ``spec``. This package will depend on:
+
+- The Move code being verified
+- Certora’s ``cvlm`` specification library
+- Platform-specific summary packages (e.g., for Sui)
+
+Below is an example ``Move.toml`` for a spec package for the
+`Review Rating <https://docs.sui.io/guides/developer/app-examples/reviews-rating>`_
+example from Sui’s documentation:
+
+.. code-block:: toml
+
+    [package]
+    name = "spec"
+    edition = "2024.beta"
+
+    [dependencies]
+    reviews_rating = { local = "../sui/examples/move/reviews_rating" }
+    cvlm = { git = "https://github.com/Certora/cvl-move-proto.git", subdir = "cvlm", rev = "main" }
+    certora_sui_summaries = { git = "https://github.com/Certora/cvl-move-proto.git", subdir = "certora_sui_summaries",  rev = "main" }
+
+    [addresses]
+    spec = "0x0"
+
+The three dependencies are:
+
+* ``reviews_rating`` – the code being verified  
+* ``cvlm`` – the Move specification language providing rules, summaries, etc.  
+* ``certora_sui_summaries`` – common summaries for Sui (storage, events, types, etc.)
+
+Writing a Rule
+--------------
+
+Rules are Move functions. A module declares its rules in a special function
+called ``cvlm_manifest``. For example:
+
+.. code-block:: rust
+
+    module spec::rules;
+
+    use reviews_rating::service::{ Service, ProofOfExperience };
+    use sui::clock::Clock;
+    use std::string::String;
+
+    use cvlm::asserts::cvlm_satisfy;
+    use cvlm::manifest::rule;
+
+    public fun cvlm_manifest() {
+        rule(b"write_new_review_sanity");
+    }
+
+    public fun write_new_review_sanity(
+        service: &mut Service,
+        owner: address,
+        content: String,
+        overall_rate: u8,
+        clock: &Clock,
+        poe: ProofOfExperience,
+        ctx: &mut TxContext,
+    ) {
+        service.write_new_review(owner, content, overall_rate, clock, poe, ctx);
+        cvlm_satisfy_msg(true, b"Reached end of function");
+    }
+
+Key points:
+
+* ``cvlm_manifest`` registers rules in the module.  
+* When the prover runs a rule, **all parameters are nondeterministically instantiated**.  
+* ``cvlm_satisfy`` creates a *satisfy rule*: it asks the prover to explore whether an execution satisfying the condition exists.
+* ``cvlm_assert`` and other CVLM constructs may also be used.
+
+The MoveProver can also automatically generate sanity rules using
```

Comment:
Sometimes you say `Prover`, sometimes `MoveProver`, and the command is `certoraSuiProver`. Let's choose one term and stick with it. My vote is for `SuiProver` to be consistent with the cli (and assuming we will potentially have a separate `certoraAptosProver` in the future with separate documentation).